scripts/pi/firstboot.sh

```diff
@@ -55,7 +55,7 @@ apt -y update
 #apt -y upgrade
 apt -y --autoremove purge ifupdown dhcpcd5 isc-dhcp-client isc-dhcp-common rsyslog avahi-daemon
 apt-mark hold ifupdown dhcpcd5 isc-dhcp-client isc-dhcp-common rsyslog raspberrypi-net-mods openresolv avahi-daemon libnss-mdns
-apt -y install libnss-resolve hostapd dnsmasq samba git python3 python3-pip nginx openssh-server
+apt -y install libnss-resolve hostapd dnsmasq dnsutils samba git python3 python3-pip nginx openssh-server
```

Comment:
`dnsutils` is useful on the raspberrypi as it contains `dig` and `host` dns lookup commands. If needing to debug dns setup this would be extremely helpful to include.